examples/with-svelte/packages/ui/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "ui",
+  "version": "0.0.0",
+  "description": "Styles and components for use in svelte websites",
+  "type": "module",
+  "module": "index.ts",
+  "svelte": "index.ts",
+  "license": "MIT",
+  "devDependencies": {
+    "svelte": "^3.9.2"
+  },
+  "files": [
```

Comment:
This is going to troll people if the package is ever published.

Reply:
(Or add a readme that gives a heads up about the `files` property in `package.json`)

Reply:
Yea this is fair, I'll just remove it.